opus/application/static_media/js/browse.js

```diff
@@ -1209,21 +1185,88 @@ var o_browse = {
         return url;
     },
 
+    // return the infiniteScroll container class for either gallery or table view
+    // NOTE: the reason we don't want to use a single unified class is because we have two infiniteScroll instances.
+    // If they share a same class say "op-scroll-container", then later on $(selector).infiniteScroll("loadNextPage") will call load event handler twice (selector selects both container) and render the data twice (duplicated data).
+    getScrollContainerClass: function() {
+        return (opus.prefs.browse === "gallery" ? ".op-gallery-view" : ".dataTable");
+    },
+
+    // Instantiate infiniteScroll
+    initInfiniteScroll: function(selector) {
+        let view = o_browse.getViewInfo();
```

Comment:
suggestion that we stop using getViewInfo and use opus.prefs directly; example:
            let startobsLabel = (opus.prefs.view === "cart" ? "cart_startobs" : "startobs");
            let tab = `#${opus.prefs.view}`;
            if (opus.prefs[startobsLabel] > 0) {